homeassistant/components/mychevy/sensor.py

```diff
@@ -159,7 +159,10 @@ def name(self):
     def async_update_callback(self):
         """Update state."""
         if self._car is not None:
-            self._state = getattr(self._car, self._attr, None)
+            if self._unit_of_measurement == "miles":
+                self._state = round(getattr(self._car, self._attr, None))
```

Comment:
Would be slightly less code to just round self._state

Reply:
thanks for suggestion, code is smaller now